### PR TITLE
Copy tweaks

### PR DESCRIPTION
**Pull request by benfoxall**
# Reword `/About` page

Some of the about page wasn't relevant anymore.  I've replaced it with an outline of the day so people know what to expect, and how to join in.

## CoC links

Now link to our slack profiles 

## Embedded Podcast page

💸 🎵 

image

### CODE REVIEW

File: about.md [hunk 1]
```diff
@@ -2,40 +2,74 @@
...
 
-## Code of Conduct
+## 12.30pm, Lunch
 
-All conferences and events should be friendly, safe and inclusive spaces for anyone to attend, and Remote Hack is no different. All organisers and attendees must abide by the [Berlin Code of Conduct](https://berlincodeofconduct.org).
+Grab something to eat, join everyone on Zoom if you'd like.
+
+## 1pm → 4.30pm
+
+More hack time - finish of things, or do something entirely different
```
Comment (lpmi-13) [inline about.md:35]: Finish off

File: podcast.md [hunk 2]
```diff
@@ -17,3 +17,5 @@ Enter this feed url in your Podcast App:
 Or, subscribe on Spotify:
 
 → [AirQuotes The Podcast AirQuotes](https://open.spotify.com/show/2ppoyoTxxSv9IvRGGqLWP7?si=tuWMxyvWShO9OlcOvljgmg)
+
+<iframe src="https://open.spotify.com/embed-podcast/show/2ppoyoTxxSv9IvRGGqLWP7" width="100%" height="232" frameborder="0" allowtransparency="true" allow="encrypted-media"></iframe>
```
Comment (lpmi-13) [inline podcast.md:21]: This might do with a `title` for a11y reasons

File: about.md [hunk 3]
```diff
@@ -2,40 +2,74 @@
...
+## Can I attend?
 
-Remote Hack helps developers and technologists get together to create solutions to Remote working, collaboration, and socialising. To go full-meta, we'll be hacking on tools for Remote hacking.
+Yes.
 
-As social distancing sets in and many of us switch to full-time Remote, this event is a chance to hack on things that make Remote a little easier.
+Even if you've never been to a hack day, or aren't tech-focused, you're absolutely welcome.
 
-## What isn't it?
+# How it works
```
Comment (lpmi-13) [inline about.md:11]: Could this be `##` or put at the top of the page? Seems to be a bit visual heirarchy clash

Comment (benfoxall) [reply about.md:11]: Great shout. Couldn't quite put my finger on why it was awkward.  I've raised the "Can I Attend" section to a `#` to it's the same level as How it Works.  Added an `hr` too to separate it out a bit.

Feel free to change it!

image

File: podcast.md [hunk 4]
```diff
@@ -17,3 +17,5 @@ Enter this feed url in your Podcast App:
 Or, subscribe on Spotify:
 
 → [AirQuotes The Podcast AirQuotes](https://open.spotify.com/show/2ppoyoTxxSv9IvRGGqLWP7?si=tuWMxyvWShO9OlcOvljgmg)
+
+<iframe src="https://open.spotify.com/embed-podcast/show/2ppoyoTxxSv9IvRGGqLWP7" width="100%" height="232" frameborder="0" allowtransparency="true" allow="encrypted-media" title="Spotify Podcast player - AirQuotes The Podcast AirQuotes"></iframe>
```
Comment (spikeheap) [inline podcast.md:21]: 🎧 ❤️

File: code-of-conduct.md [hunk 5]
```diff
@@ -4,10 +4,10 @@ layout: default
...
 For any issues/questions/queries, please contact an organiser:
 
-|             | Slack | Twitter                                                |
-| ----------- | ----- | ------------------------------------------------------ |
-| Ben Foxall  | @ben  | [@benjaminbenben](https://twitter.com/@benjaminbenben) |
-| Ryan Brooks | @ryan | [@spikeheap](https://twitter.com/@spikeheap)           |
+|             | Slack                                                  | Twitter                                                |
+| ----------- | ------------------------------------------------------ | ------------------------------------------------------ |
+| Ben Foxall  | [Ben](https://remote-hack.slack.com/team/U010PB5GB4N)  | [@benjaminbenben](https://twitter.com/@benjaminbenben) |
+| Ryan Brooks | [Ryan](https://remote-hack.slack.com/team/U010MNGLJ1K) | [@spikeheap](https://twitter.com/@spikeheap)           |
```
Comment (spikeheap) [inline code-of-conduct.md:10]: ❤️ much better

**Pull request merged**